views/proposal.tmpl

```diff
@@ -41,8 +41,8 @@
                 <span class="fs18 font-weight-bold lh1rem">Vote scheduled</span>
                 <div class="py-2">
                   {{.Username}} has authorized voting on this proposal.
-                  Voting begins at block {{$metadata.StartHeight}}
-                  (about <span class="medium-sans">{{secondsToShortDurationString $metadata.SecondsTil}}</span>)
+                  Waiting for the Politeia admin to approve the vote start.
+                  (should last for <span class="medium-sans">{{secondsToShortDurationString $metadata.SecondsTil}}</span>)
```

Comment:
How about "Awaiting admin approval (vote duration: 7 days)" or even just "Awaiting admin approval"? Drop the period as well.

`SecondsTil` is not the right field for this value. It's a constant of 7 days here, so we don't actually need a variable or even to list it.